test/tests/morebits.batchOperation.js

```diff
@@ -0,0 +1,27 @@
+QUnit.module('Morebits.batchOperation');
+var batch = new Morebits.batchOperation();
+QUnit.test('Contruction', assert => {
+	assert.true(batch instanceof Morebits.batchOperation, 'Correct instance');
+});
+var pageList = ['Macbeth', 'Hamlet', 'Romeo and Juliet', 'Much Ado About Nothing', 'Tempest'];
+batch.setPageList(pageList);
+var chunkLength = 2;
+batch.setOption('chunkSize', chunkLength);
+QUnit.test('Run', assert => {
+	var all = '';
+	var runFunc = function(page) {
+		all += page;
+		batch.workerSuccess();
+	};
+	batch.run(runFunc);
+	assert.equal(all, pageList.join(''), 'batch run');
```

Comment:
`assert.equal`, here and everywhere, does comparisons with `==`. Should be avoided in favour of `assert.strictEqual`. This is why Qunit is annoying.

Reply:
Done!